ui/packages/consul-ui/app/components/consul/service/search-bar/index.hbs

```diff
@@ -140,26 +140,8 @@ as |key value|}}
         </BlockSlot>
         <BlockSlot @name="options">
   {{#let components.Optgroup components.Option as |Optgroup Option|}}
-{{#let
-  (reject-by 'Partition' @partition @partitions)
-as |nonDefaultPartitions|}}
-{{#if (gt nonDefaultPartitions.length 0)}}
-    <Optgroup
-      @label={{t 'common.brand.consul'}}
-    >
-    {{#each @partitions as |partition|}}
-      <Option class="partition" @value={{partition}} @selected={{includes partition @filter.source.value}}>
-        {{partition}}
-      </Option>
-    {{/each}}
-    </Optgroup>
-{{/if}}
-{{/let}}
-
 {{#if (gt @sources.length 0)}}
-    <Optgroup
-      @label={{t 'common.search.integrations'}}
-    >
+    <Optgroup>
```

Comment:
Pretty sure you can just remove the optgroup here, which will mean the amend made underneath in this PR in `ui/packages/consul-ui/app/components/popover-select/optgroup/index.hbs` isn't required. Optgroups are only really used to provide a label to the group, otherwise they are unnecessary.